Include/cpython/pystate.h

```diff
@@ -29,6 +29,14 @@ typedef int (*Py_tracefunc)(PyObject *, PyFrameObject *, int, PyObject *);
 #define PyTrace_C_RETURN 6
 #define PyTrace_OPCODE 7
 
+/* Remote debugger support */
+# define MAX_SCRIPT_PATH_SIZE 512
+typedef struct _remote_debugger_support {
+    int enabled;
+    int debugger_pending_call;
```

Comment:
We should probably be explicit about our sizes:
```suggestion
    int32_t enabled;
    int32_t debugger_pending_call;
```
Granted `int` is 32 bit on all modern platforms, but given that this goes into the offsets and needs to be read from another process with potentially different integer types (like a 64-bit process attaching to a 32-bit one or vice versa) it's better to be explicit, I think.

Or, if you'd prefer, the offsets themselves could contain the size of the field - but I think that's overkill.